torch/lib/c10d/ProcessGroupMPI.hpp

```diff
@@ -77,24 +81,31 @@ class ProcessGroupMPI : public ProcessGroup {
   class WorkMPI : public ProcessGroup::Work {
    public:
     WorkMPI(
+        const std::shared_ptr<std::vector<at::Tensor>> outputs,
```

Comment:
Is it intentional to have outputs/inputTensors instead of ouputTensors/inputTensors?